src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        final int Index = new Random().nextInt(Color.values().length);

```

Comment:
make random a class field - don't recreate per each method invocation, as you do now
variables names should not be capitalized